README.rst

```diff
@@ -192,8 +192,8 @@ Prerequisites
    +----------------------------------------+---------------------------------------------------------------------------------------------+
    | |ffmpeg link|_ or later                | We recommend using version 3.4.2 compiled following the *instructions below*.               |
    +----------------------------------------+---------------------------------------------------------------------------------------------+
-   | |opencv link|_ or later                | | We recommend using version 3.4+. However previous versions are also compatible.           |
-   |                                        | | *OpenCV 2.x compatibility is provided unofficially.*                                      |
+   | |opencv link|_ or later                | | We recommend using version 3.4+.                                                          |
+   |                                        | | Older versions from the 3.X family are also supported.                                    |
```

Comment:
I'd see few things here (we might as well decide on them now).
Firstly, I'd paraphrase it 
```
Minimum version: 3.0
Recommended version: 3.4
```
But now `3.4+` means also `4.0` and we don't actually test it.
Also I think it should be clear here, why there is a recommended version. Is it because of performance? Or some features wouldn't be present? In my opinion, user has to be aware of such limitations, when reading this doc.

On the other hand, we may just `Supported version: 3.4`. This resolves all ambiguities and clearly states, that we test against 3.4 only...

Reply:
I would say only
```
Recommended version: 3.4
```
Other that this is up to the user. It will probably work with 3.0 (unless we add come incompatibility) and 4.0 unless 4.0 changes something - but we are not testing that and there is no guarantee.

Reply:
> [...] but we are not **testing** that and there is no **guarantee**.

That's what I would call `supporting` ;) After all `recommended != supported`...

`Supported version: 3.4`?

Reply:
#735 - it turns out, that 4.0 changes something and doesn't work. Maybe we can write `Supported version 3.4` then?